### PR TITLE
Show submitted forms in backpack and allow editing existing submissions

### PR DESCRIPTION
**Pull request by narendrasss**
Implements #220 and #225

**Changes**

* Display latest user entries in backpack
* Add entries endpoint to fetch form submissions for that particular user
* Move `log` computation logic to a `vuex` `logs` getter function
* Add `entries` as a property of each question

### CODE REVIEW

**Comment by wynnset** [comment]
@anhduc130 as discussed, please fix this branch to add the completed h5p audio back into the backpack.

**Comment by wynnset** [comment]
This is not behaving as expected.

**Expected behavior:** There is only 1 entry in the backpack for each form entry.
**Actual behavior:** The backpack is showing multiple rows with the same entry (2 rows for text and 3 rows for checklist).

**Steps to reproduce:**
1. Logged in as admin, add 2 question set nodes to a stage
2. For each of the question sets, add a question
3. For each question added, add both checklist and a text entry answer type
4. Logout and login as a user who is not an editor/admin
5. Answer one of the questions using the text entry and the other one using a checklist
6. Open the backpack to see the duplicate rows

**Comment by wynnset** [comment]
One more problem: The thumbnails are not working anymore. Please make sure the thumbnails show similar to how they are currently showing in the `tyde` branch (thumbnail in the back with icon over-laid).

**Comment by wynnset** [comment]
Problem #3: If the user has line breaks in their answers, the line breaks should be displayed in the backpack like they were entered by the user. It currently is not showing any line breaks.

**Comment by narendrasss** [comment]
@wynnset I can't seem to replicate the bug with the duplicate entries, but I fixed the other two errors.

**Comment by narendrasss** [comment]
@wynnset I fixed a small issue with the menu crashing, but I tested it again and no duplicates. Everything else seems to work as it should.

File: endpoints.php [hunk 1]
```diff
@@ -744,6 +750,23 @@ function updateTapestryNodeCoordinates($request)
     }
 }
 
+/**
```
[the remark this answers is not in the document]
Comment (anhduc130) [reply endpoints.php]: The comments appear empty here. Could we please add some comments for this method?

**Pull request merged**